frontend/src/components/material/ScheduleEntryLinks.vue

```diff
@@ -6,9 +6,7 @@
     class="e-title-link tabular-nums"
```

Comment:
Not in this file but somewhere here:
You also need to adapt the logic when generating the Excel Download for the Materiallist

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/4811/commits/409560022d65676ea394fdf152828d0047f16846